css/style.css

```
@@ -58,22 +87,69 @@ a {
   opacity: 0.3;
 }
 
+.container {
+  width: 1200px;
```

благодаря этому появится базовая адаптивность. Контейнерам лучше не задавать жестко width совсем, лучше max-width 
```suggestion
  max-width: 1200px;
```